app/app.go

```diff
@@ -103,6 +103,16 @@ func (a *App) InitServer() {
 		if a.IsLeader() {
 			a.srv.dndTask = model.CreateRecurringTask("Unset DND Statuses", a.UpdateDNDStatusOfUsers, 300*time.Second)
 		}
+		a.srv.AddClusterLeaderChangedListener(func() {
```

Comment:
This should be done in the `app/server.go` I think.

Reply:
in app/server.go we don't have App receiver so it won't be possible to start a task for App.UpdateDNDStatusOfUsers